composer.json

```diff
@@ -16,13 +16,16 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.5.9|~7.0",
         "symfony/framework-bundle": "~2.8|~3.0",
         "symfony/icu": "~1.1",
         "swiftmailer/swiftmailer": ">=4.1.3,~5.0",
         "doctrine/orm": "~2.4",
         "twig/twig": "~1.3"
     },
+    "suggest": {
+        "sonata-project/admin-bundle": "The missing Symfony Admin Generator; require for the frontend"
```

Comment:
> require for the frontend

By default Sonata admin is not required for this bundle, we provide a simple bootstrap admin. 
You will need Sonata only if you enable it:
```yml
lexik_mailer:
    sonata_integration: true # false by default
```